.pre-commit-config.yaml

```
@@ -28,6 +28,7 @@ repos:
       - id: ruff
         name: ruff-doc
         files: doc/data/messages
+        exclude: doc/data/messages/u/unreachable-match-patterns/bad.py
```

```suggestion
        # Please exclude using doc/data/ruff.toml
        # exclude: ""  # Leave empty
```
It's better as there are a lot of exclude and one place is better than two.